realm-annotations-processor/src/main/java/io/realm/processor/RealmProxyClassGenerator.java

```diff
@@ -155,22 +156,42 @@ public void generate() throws IOException, UnsupportedOperationException {
 
         writer.emitPackage(REALM_PACKAGE_NAME)
                 .emitEmptyLine();
-        writer.emitImports(
-                "android.util.JsonReader",
-                "android.util.JsonToken",
-                "io.realm.RealmObject",
-                "io.realm.internal.ColumnType",
-                "io.realm.internal.Table",
-                "io.realm.internal.ImplicitTransaction",
-                "io.realm.internal.LinkView",
-                "io.realm.internal.android.JsonUtils",
-                "java.io.IOException",
-                "java.util.*",
-                "org.json.JSONObject",
-                "org.json.JSONException",
-                "org.json.JSONArray",
-                packageName + ".*")
-                .emitEmptyLine();
+
+        ArrayList<String> imports = new ArrayList<String>();
+        imports.add("android.util.JsonReader");
+        imports.add("android.util.JsonToken");
+        imports.add("io.realm.RealmObject");
+        imports.add("io.realm.internal.ColumnType");
+        imports.add("io.realm.internal.Table");
+        imports.add("io.realm.internal.ImplicitTransaction");
+        imports.add("io.realm.internal.LinkView");
+        imports.add("io.realm.internal.android.JsonUtils");
```

Comment:
io.realm.internal should be rearrange so it is in alphabetical order. IDE's does this automatically and it is a pain when editing the unit tests if they are not in alfabetical order.